src/logic/safe/store/selectors/gatewayTransactions.ts

```diff
@@ -127,3 +127,23 @@ export const getTransactionsByNonce = createSelector(
     return txsByNonce
   },
 )
+
+export const getLastTransaction = createSelector(nextTransactions, queuedTransactions, (nextTxs, queuedTxs) => {
```

Comment:
Shouldn't it also search in the history?

Reply:
I had included the history transactions in the selector initially but then removed it as it was not necessary for the current uses.
Included it again after this comment as it turns the selector more useful.